contracts/contracts/RaidenMicroTransferChannels.sol

```diff
@@ -344,6 +352,7 @@ contract RaidenMicroTransferChannels {
 
         require(channels[key].deposit > 0);
         require(closing_requests[key].settle_block_number == 0);
+        require(channels[key].deposit + _added_deposit <= channel_deposit_bugbounty_limit);
```

Comment:
that's overflow, no? If I test this, I can submit `_added_deposit=2**192-1` and with a deposit of just 1 it doesn't fail.
```
    function addall() {
        uint192 add1 = 10;
        uint192 add2 = 2**192-1;
        require(add1 + add2 <= 100);
    }
```

Reply:
If it is an overflow, it will fail at `assert(channels[key].deposit > _added_deposit);` , right?
So you tested and it did not fail?

Reply:
yes, that's right, it would fail then.